exercism-modern.el

```diff
@@ -245,13 +245,12 @@ Pass prefix BUFFER-PREFIX-ARG to prompt for a buffer instead."
                                            (blurb (alist-get 'blurb exercise))
                                            (difficulty (alist-get 'difficulty exercise))
                                            (is-unlocked (not (eq :json-false (alist-get 'is_unlocked exercise))))
-                                           (is-recommended (not (eq :json-false (alist-get 'is_recommended exercise))))
```

Comment:
Why removed?

Reply:
Because the linter said it was unused? (and because it was not used anywhere else)

Reply:
Oh that's a bug, the star should only be shown for recommended exercises

Reply:
Resolved in 4cdb41c208dda18386f383fbf0333ea7155463d2